Fix an issue where Vercel adapter may create functions for prerendered routes (#10231)
withastro/astro: .changeset/four-shoes-rule.md

---
"@astrojs/vercel": patch
---

> Fixes an issue where Vercel adapter may create functions for prerendered routes

Reviewer: ```suggestion
Fixes an issue where functions were also created for prerendered routes with `functionPerRoute` enabled.
```
